py/entry_objects.py

```diff
@@ -0,0 +1,75 @@
+class UpcomingEvent:
+    """
+    Represents one entry of an upcoming event
+    Attributes:
+    1. title (string) - The title of the event
+    2. details (array of link or text objects) - Each element in the array represents a new line
+    3. body (array of line or text objects) - Each element in the array represents a new line
+    """
+    def __init__(self, title, details, body):
+        self.title = title
+        self.details = details
+        self.body = body
+    
+class Entry:
+    """
+    Represents one entry of all other sections other than upcoming events
+    Attributes:
+    1. title (string) - The title of the entry
+    2. body (array of line, text or image objects) - Each element in the array represents a new line
+    """
+    def __init__(self, title, body):
+        self.title = title
+        self.body = body
+
+class Milestones(Entry):
+    """
+    Inherits from Entry class and represents one milestone entry.
+    """
+    def __init__(self, title, body):
+        super().__init__(title, body)
+
+class ExecUpdates(Entry):
+    """
+    Inherits from Entry class and represents one milestone entry.
+    """
+    def __init__(self, title, body):
+        super().__init__(title, body)
+
+class AlumniAdvice(Entry):
+    """
+    Inherits from Entry class and represents one milestone entry.
+    """
+    def __init__(self, title, body):
+        super().__init__(title, body)
+        
+class Text:
+    """
+    Represents a portion of text of an entry's body.
+    Attributes:
+    1. text (string) - The actual text that needs to be displayed
+    2. linebreaks (integer) - The number of empty lines that needs to follow the text
```

Comment:
Having linebreaks as an attribute feels odd to me, wouldn't people just add "\n\n\n\n\n..." at the end of their strings to add more space? Even so, I feel like this is still bad practice to just slap linebreaks to do spacing when we should be customizing css - HTML is meant to determine structure of the contents of a document, while CSS determines how the contents are arranged. Having linebreaks appended at the end breaks this principle.